source/Variant.cpp

```diff
@@ -0,0 +1,171 @@
+/* Variant.cpp
+Copyright (c) 2019 by Michael Zahniser
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#include "Fleet.h"
+
+#include "DataNode.h"
+#include "GameData.h"
+#include "Random.h"
+#include "Ship.h"
+
+#include <algorithm>
+#include <cmath>
+#include <iterator>
+
+using namespace std;
+
+namespace {
+}
```

Comment:
Yeah, that's just a holdover from copy-pasting the file structure from another file and removing what wasn't needed. This will either be used or removed by the time I mark this PR ready for review.